module/VuFind/src/VuFind/RecordDriver/MarcReaderTrait.php

```diff
@@ -58,7 +58,12 @@ trait MarcReaderTrait
     public function getMarcRecord()
     {
         if (null === $this->lazyMarcRecord) {
-            $marc = trim($this->fields['fullrecord']);
+            // Get preferred MARC field from config, if it is set and is existing:
+            $marcField = (isset($this->mainConfig->Record->preferredMarcField)
```

Comment:
I think this might be a little more readable in two lines as:

<pre>
$preferredMarcField = $this->mainConfig->Record->preferredMarcField ?? 'fullrecord';
$marc = trim($this->fields[$preferredMarcField] ?? $this->fields['fullrecord']);
</pre>

Reply:
Agreed. I will incorporate this in the next version.